docs/blog/2025-january-update.rst

```diff
@@ -0,0 +1,59 @@
+.. post:: Jan 7, 2025
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - January 2025
+===============================================
+
+Hey folks, Eric Holscher here, one of the co-founders of Write the Docs.
+I'm here with our traditional January community update while the newsletter team takes the month off.
+
+2025 is going be mostly a continuation of our efforts from last year.
+We don't have any major new initiatives planned,
+but we continuing to refine our approach to conferences and building community.
+
+2024 Salary Survey ends soon
+----------------------------
+
+The `2024 Salary Survey <https://salary-survey.writethedocs.org/>`_ is still open until January 15.
+We've only had 636 responses so far,
+which is quite a bit smaller than last year,
+so please take a moment to fill it out or share it with a friend if you haven't already.
+We plan to release the results in Q2, in time for the Portland conference.
+
+Our 2025 conferences
+--------------------
+
+It was wonderful to see everyone at our 2024 Portland, Atlantic, and Australia conferences.
+Our 2025 conferences will look similar to last year:
+
+* `Portland <https://www.writethedocs.org/conf/portland/2025/>`_ will be in-person on **May 4-6**.
+* **Atlantic** will be a virtual event, spanning the East Coast and European timezones. We might push it back a bit to avoid the many conferences that happen in mid Septermber, but haven't firmed up dates.
+* Australia just happened a month ago, and we haven't started planning for 2025 yet.
+
+We are currently accepting `talk proposals <https://www.writethedocs.org/conf/portland/2025/cfp/>`_ and selling tickets for our Portland conference.
```

Comment:
Not sure why this text is separated from the bullet item for Portland, I'd have expected to have it in that item but feel free to overrule if you prefer.

Reply:
Mostly just has more text with it, so didn't want to overload the bullets.